packages/features/ee/teams/components/MemberList.tsx

```diff
@@ -454,99 +462,8 @@ export default function MemberList(props: Props) {
-                                  {t("impersonate")}
-                                </DropdownItem>
-                              </DropdownMenuItem>
-                              <DropdownMenuSeparator />
-                            </>
-                          )}
-                          {resendInvitation && (
-                            <DropdownMenuItem>
-                              <DropdownItem
-                                type="button"
-                                onClick={() => {
-                                  resendInvitationMutation.mutate({
-                                    teamId: props.team?.id,
-                                    email: user.email,
-                                    language: i18n.language,
-                                  });
-                                }}
-                                StartIcon="send">
-                                {t("resend_invitation")}
-                              </DropdownItem>
-                            </DropdownMenuItem>
-                          )}
-                          <DropdownMenuItem>
-                            <DropdownItem
-                              type="button"
-                              onClick={() =>
-                                dispatch({
-                                  type: "SET_DELETE_ID",
-                                  payload: {
-                                    user,
-                                    showModal: true,
-                                  },
-                                })
-                              }
-                              color="destructive"
-                              StartIcon="user-x">
-                              {t("remove")}
-                            </DropdownItem>
-                          </DropdownMenuItem>
-                        </DropdownMenuContent>
-                      </Dropdown>
-                    )}
-                  </ButtonGroup>
-                  <div className="flex md:hidden">
-                    <Dropdown>
-                      <DropdownMenuTrigger asChild>
-                        <Button type="button" variant="icon" color="minimal" StartIcon="ellipsis" />
-                      </DropdownMenuTrigger>
-                      <DropdownMenuContent>
-                        <DropdownMenuItem className="outline-none">
-                          <DropdownItem
-                            disabled={!user.accepted}
-                            href={!user.accepted ? undefined : `/${user.username}`}
-                            target="_blank"
-                            type="button"
-                            StartIcon="external-link">
-                            {t("view_public_page")}
-                          </DropdownItem>
-                        </DropdownMenuItem>
-                        {editMode && (
-                          <>
+                        <DropdownMenuPortal>
```

Comment:
It's just wrapped by DropdownMenuPortal. No actual change. 

![Screenshot 2024-12-02 at 15 12 47](https://github.com/user-attachments/assets/8def2798-f2cd-4b34-94c8-039a63d3adcb)